test/completion.js

```diff
@@ -460,4 +460,21 @@ describe('Completion', () => {
     r.errors.length.should.equal(0)
     r.logs.should.include('--foo:bar')
   })
+  it('when using subcommands ensure early bailout if full command is typed (zsh)', () => {
```

Comment:
rather than the `(zsh)` in the description, let's move the `zsh` tests into their own describe:

```js
describe('zsh', () => {
  it('exits early if full command is typed')
})
```

Reply:
@bcoe took a stab at this (see https://github.com/trevorlinton/yargs/pull/1).  Here is a screenshot of logs when specs are run:
<img width="678" alt="screen shot 2019-02-15 at 8 31 17 pm" src="https://user-images.githubusercontent.com/3240137/52892573-a321c980-3161-11e9-997b-eefddcc2ca61.png">